package.json

```diff
@@ -103,6 +103,7 @@
     "jest-html-reporter": "^3.3.0",
     "jest-image-snapshot": "^4.2.0",
     "jest-puppeteer": "^4.4.0",
+    "jest-sonar": "^0.2.11",
```

Comment:
:question: do you know if this has perf impact on our local dev i.e if you have seen processing time increase when running tests?
I will do some checks on my side

Reply:
I didn't check yet.

Reply:
## Ubuntu
### Before jest-sonar integration
![image](https://user-images.githubusercontent.com/4921914/100436303-76714480-309f-11eb-82e6-02f6ee9d6597.png)

### After jest-sonar integration
![image](https://user-images.githubusercontent.com/4921914/100436604-dbc53580-309f-11eb-918f-88303c893296.png)

## MacOS
### Before jest-sonar integration
![image](https://user-images.githubusercontent.com/4921914/100436475-b506ff00-309f-11eb-98ed-e0eacf24f26d.png)

### After jest-sonar integration
![image](https://user-images.githubusercontent.com/4921914/100436649-ebdd1500-309f-11eb-830b-12df259bee7b.png)


## Windows
### Before jest-sonar integration
![image](https://user-images.githubusercontent.com/4921914/100436515-bf28fd80-309f-11eb-89b3-f3867af2622e.png)

### After jest-sonar integration
![image](https://user-images.githubusercontent.com/4921914/100436689-f9929a80-309f-11eb-8075-969c2dd26713.png)

Reply:
Ok, thanks for the figures
It looks ok